src/js/stationGenerator.js

```diff
@@ -278,6 +374,8 @@ export function getCallingStation() {
     ),
     name: randomElement(names),
     state: isUS ? randomElement(stateAbbreviations) : '',
+    section: randomElement(arrlSections), // would be nice to pick from a map of state to random section in that state but would miss canada and we use only one or the other anyway
+    klass: (Math.floor(Math.random() * 20) + 1).toString() + randomElement(fieldDayClasses),
```

Comment:
weighting this toward the low end would be more helpful. There are a lot of `1*` "groups" and exponentially fewer higher station count groups. If you can easily copy the whole `1D` or `1E`, that will get you a lot farther than being good at copying `20F`.

Reply:
I really wondered about that. But then you might equally argue that the sections should be weighted by popularity, and the names used in some exchanges should be weighted by frequency. There's the argument that the point of the simulation is not to be precise, but to expose you to variations that you might not at first see when you are actually operating, so you are better off learning that the class can go as high as 20 in the simulation even though you will see it less in real life.

Reply:
x² helps make the lower numbers more likely, and that's implemented now.